public/usage-examples/terminal/write/write-1-loading-bar.txt

```diff
@@ -0,0 +1,3 @@
+### Loading Bar
```

Comment:
Because these have 3 # they are showing as links in the side bar as such: 
![image](https://github.com/user-attachments/assets/3e65f840-1d1d-4298-a380-c74a0429d29f)

I think this should have 4 # to be a sub heading with the usage example link being the name of the function.

Reply:
I think this is something that would have to be modified in the component.cjs
As all usage examples should be ###